meson.build

```diff
@@ -325,6 +325,17 @@ if not get_option('mxnet-support').disabled()
   endif
 endif
 
+# ml-agentd
+ml_agentd_dep = dependency('', required: false)
```

Comment:
Why do you need this much complicated? Don't we have pkgconfig enabled for magenta?

Reply:
@myungjoo mlagent provides pkgconfig thus ``` ml_agentd_dep = dependency('ml-agentd', required: false)``` is enough for most cases I guess. The other lines are only for providing fallback mechanisms to support the cases somehow the dependency method could not resolve although ```true``` (or ```auto```) has been given.